src/Codeception/Lib/ModuleContainer.php

```diff
@@ -201,6 +201,42 @@ public function validateConflicts()
         }
     }
 
+    /**
+     * @return string
+     */
+    public function getCurrentEnvironment()
+    {
+        if (empty($this->config['current_environment'])) {
```

Comment:
fair enough, it now throws an exception when the `current_environment` key is not available.